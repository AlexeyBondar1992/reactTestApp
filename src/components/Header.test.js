import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Header from './Header';

describe('Card', () => {
    it('should test component creation', () => {
        expect(shallow(<Header />).length).toEqual(1);
    });

    it('should test snapshot to match', () => {
        const wrapper = shallow(<Header />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
