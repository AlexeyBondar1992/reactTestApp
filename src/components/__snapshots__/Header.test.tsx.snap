// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should test snapshot to match 1`] = `
<div>
  <h1
    className="f1"
  >
    RoboFriends
  </h1>
  <CounterButton
    color="red"
  />
</div>
`;
